Source/WebKit/Shared/IPCStreamTester.cpp

```diff
@@ -93,6 +93,12 @@ void IPCStreamTester::syncMessageReturningSharedMemory1(uint32_t byteCount, Comp
     completionHandler(WTFMove(result));
 }
 
+void IPCStreamTester::syncMessageEmptyReply(uint32_t value, CompletionHandler<void()>&& completionHandler)
+{
+    UNUSED_PARAM(value);
```

Comment:
I need value so I can trigger a deserialisation error when it's excluded

Reply:
You need this function to have a uint32_t parameter, but you don't need to name it "value"
...(uint32_t, CompletionHandler<...